### PR TITLE
grammer, spelling, and style edits

### PR DESCRIPTION
**Pull request by erikfriesen**
Wasn't sure if we needed to replace "Boundless" with "Planet Federal". Left them alone for now.

### CODE REVIEW

File: docs/source/components/qgis.rst [hunk 1]
```diff
@@ -28,14 +28,13 @@ processing providers are available for QGIS’s Processing framework:
 Quickstart guide
 ----------------
 
-This section aims to give you a brief overview of QGIS. It will mainly focus
-on QGIS's graphical user interface (GUI) and some of the most common operations
-like loading data, managing layers and printing a map. For more detailed
-information on QGIS usage see the tutorials on our learning center and
-consult :qgis_docs:`QGIS's official documentation <2.18>`.
+This brief overview of QGIS will focus on QGIS's graphical user interface (GUI)
+and some of the most common operations like loading data, managing layers and
```
Comment (dakcarto) [inline docs/source/components/qgis.rst]: "and some common operations ..."

File: docs/source/install/uninstall.rst [hunk 2]
```diff
@@ -8,34 +8,31 @@ Uninstalling Boundless Desktop
 Windows
 -------
 
-In windows, to uninstall Boundless Desktop |version| core apps, libs and
-utilities and the symbolically linked applications and documentation, use the
-`Uninstall Desktop` shortcut available in the Boundless Desktop shortcut
-folder or from the start menu.
+Use `Uninstall Desktop` in the Boundless Desktop shortcut folder onthe Desktop or in the Start Menu to
```
Comment (dakcarto) [inline docs/source/install/uninstall.rst]: onthe --> on the

**Review by dakcarto — CHANGES_REQUESTED** [review]
Here are some small changes...

File: installers/source/win/known_issues.rst [hunk 3]
```diff
@@ -1,20 +1,22 @@
...
+* Running QGIS from a pinned icon in the taskbar will prevent the execution of scripts
+  in the init_script folder.
 * Uninstaller leaves an empty ``Program Files\Boundless\Desktop\2.0`` directory
-* Uninstaller on Windows 7 will need a system reboot to allow other programs uninstall.
+* Uninstaller on Windows 7 requires a system reboot before other programs can be
+  uninstalled.
 * Uninstallation on user workspace requires administrator permissions.
-* Currently available versions of IPython console plugin (v1.9) is not compatible with
+* Currently available version of IPython console plugin (v1.9) is not compatible with
   IPython 7.2, which is the version deployed with Boundless Desktop. The plugin
```
Comment (dakcarto) [inline installers/source/win/known_issues.rst]: "... is not compatible with the IPython version deployed with Boundless Desktop."

Internal note: the IPython version is actually 7.4, but we should just skip referencing its version here, just note its incompatibility.

File: installers/source/win/version_changelog.rst [hunk 4]
```diff
@@ -23,25 +23,25 @@ based on Qt 5 and Python 3 and adds many new features. Some examples are:
...
 * Locator bar for quick access to tools and commands
-* Improved consistency to the user interface
+* Improved consistency in the user interface
 * Tens of new expression functions, including array functions
 * Improved expressions editor with auto-completion and error highlighting
 * New units for symbols and layouts, including pixels and inches
 * Point cluster renderer
 * Easy styling of discrete rasters
-* Note tool is replaced by a much improved and functional vertex tool
+* Note tool is replaced by a much improved vertex tool with greater functionality
```
Comment (dakcarto) [inline installers/source/win/version_changelog.rst]: Note -> Node

**Review by dakcarto — CHANGES_REQUESTED** [review]

File: installers/source/win/known_issues.rst (hunk 3 above)
Comment (erikfriesen) [reply installers/source/win/known_issues.rst]: whoops, i like your _specific_ suggestion better than what i just committed. redoing it

**Comment by erikfriesen** [comment]
Additional notes:
- `win/dependencies.rst` does not appear in the output currently on s3.
- Does the copyright need to be updated? (related to Boundless branding question above)

**Review by dakcarto — APPROVED** [review]

**Comment by dakcarto** [comment]
> Additional notes:
> 
> * `win/dependencies.rst` does not appear in the output currently on s3.

Hmm. Indeed.

> * Does the copyright need to be updated? (related to Boundless branding question above)

Yes... at least the year needs updated to 2019.

**Pull request merged**